README.md

```diff
@@ -29,30 +35,31 @@ bundle install
 bundle exec rspec
 ```
 
-See [bundler.io](http://bundler.io) for the full documentation.
-
-### Troubleshooting
-
-For help with common problems, see [TROUBLESHOOTING](doc/TROUBLESHOOTING.md).
-
-Still stuck? Try [filing an issue](doc/contributing/ISSUES.md).
-
-### Other questions
+### Contributing
 
-To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together. So if you'd like to contribute to Bundler, that's awesome, and we <3 you!
```

Comment:
I'm unsure about why we need this paragraph, can you explain why we need to talk about Ruby Together here?